packages/suite/src/components/wallet/CoinmarketTopPanel/index.tsx

```diff
@@ -0,0 +1,89 @@
+import * as routerActions from '@suite-actions/routerActions';
+import { MAX_WIDTH } from '@suite-constants/layout';
+import { useActions, useSelector } from '@suite-hooks';
+import { AccountLabeling } from '@suite-components';
+import { colors, Icon, variables } from '@trezor/components';
+import React from 'react';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+    display: flex;
+    width: 100%;
+    justify-content: center;
+    background: ${colors.NEUE_BG_LIGHT_GREY};
+    border-bottom: 1px solid ${colors.NEUE_STROKE_GREY};
+`;
+
+const Content = styled.div`
+    display: flex;
+    width: 100%;
+    padding: 0 32px;
+    min-height: 50px;
+    max-width: ${MAX_WIDTH};
+    justify-content: space-between;
+    align-items: center;
+
+    @media screen and (max-width: ${variables.SCREEN_SIZE.LG}) {
+        padding: 24px 16px 0px 16px;
+    }
+`;
+
+const Left = styled.div`
+    display: flex;
+    flex: 1;
+    align-items: center;
+    color: ${colors.NEUE_TYPE_LIGHT_GREY};
+    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+`;
+
+const Right = styled.div`
+    display: flex;
+    flex: 1;
+
+    justify-content: flex-end;
+`;
+
+const Back = styled.div`
+    display: flex;
+    cursor: pointer;
+    align-items: center;
+`;
+
+const StyledIcon = styled(Icon)`
+    width: 10px;
+    margin-right: 12px;
+`;
+
+const CoinmarketTopPanel = () => {
+    const { goto } = useActions({
+        goto: routerActions.goto,
+    });
+
+    const selectedAccount = useSelector(state => state.wallet.selectedAccount);
+    if (selectedAccount.status !== 'loaded') return null;
+    const { account } = selectedAccount;
+
+    return (
+        <Wrapper>
+            <Content>
+                <Left>
+                    <Back
+                        onClick={() =>
+                            goto('wallet-coinmarket-buy', {
+                                symbol: selectedAccount.params.symbol,
```

Comment:
`goto('wallet-coinmarket-buy', selectedAccount.params)` is enough, no need to create new exact same object

Reply:
lol you are right! :D fixed